File: Fruit_Basket.java
import java.util.Stack;
import java.util.Scanner;

public class Fruit_Basket{
    public static void main(String []args){
        Scanner sc = new Scanner(System.in);
        Stack <String> fruits = new Stack<>();
        
        String [] fruit = {"Apple","Orange","Mango","Guava"}; //Array of fruits

        System.out.println("Catch and eat any of these fruits: ('Apple','Orange','Mango','Guava')");
        System.out.println("How many fruits would you like to catch: ");
        int choice = sc.nextInt();

        sc.nextLine();
        for(int i = 0;i<choice;i++){ // looping to get ask how many fruits would you like to add within the array
            System.out.println("Choose a fruit to catch. Press A,O,M, or G: ");
                char ch = sc.next().toUpperCase().charAt(0);
                System.out.println("Fruit of " +(i + 1) + " of " + choice);   
                    if(ch == 'A'){fruits.push(fruit[0]);}
                        else if(ch == 'O'){fruits.push(fruit[1]);}
                            else if(ch == 'M'){fruits.push(fruit[2]);}
                                else if(ch == 'G'){fruits.push(fruit[3]);}
                else{
                    System.out.println("Invalid Input.Please choose one of the choices that has been given.");
                }
        }
        System.out.println("Your basket has: " + fruits); //Displaying the contents of the array

        for(int j = 0;j < choice;j++){ //looping function to keep removing the items within the array till its empty.
            System.out.println("Press 'E' to eat a fruit: ");
                char e = sc.next().toUpperCase().charAt(0);
                    if(e == 'E'){
                        fruits.pop();
                            if(fruits.isEmpty()){
                                System.out.println("No more Fruits.");
                         }else{
                        System.out.println("Fruit(s) in the basket: " + fruits);       
                        }
            }else{
                System.out.println("Invalid Input.Please try again.");
            }
    }
        sc.close();
    }
}

File: fruit_Basket.py
basket = []

fruit = ['Apple','Orange','Mango','Guava']
f = ['A','O','M','G']

print("Catch and eat any of these fruits: ", fruit, ": ")
catch = int(input("How Many fruits would you like to catch: "))

for i in range(catch):
    choice = input("Choose a fruit to catch. Press A, O, M, or G: ").upper()[0]
    print('Fruit ', (i + 1), 'of', catch)
    if choice == f[0]:
        basket.append(fruit[0])
    elif choice == f[1]:
        basket.append(fruit[1])
    elif choice == f[2]:
        basket.append(fruit[2])
    elif choice == f[3]:
        basket.append(fruit[4])
    elif choice == f[4]:
        basket.append(fruit[4])
    else:
        print("Invalid input.")

print("Your basket now has:")
for x in basket:
    print(x)

for j in range(catch):
    remove = input("Press 'E' to eat a fruit:").upper()
    if remove == 'E':
        basket.pop()
        if not basket:
            print("No More Fruits.")
        else:
            print("Fruit(s) in the basket: ", str(basket))
    else:
        print("Invalid Input.Please try again.")

